app-shells/fish/fish-3.7.1.recipe

```diff
@@ -25,7 +25,7 @@ PROVIDES="
 	"
 REQUIRES="
 	haiku$secondaryArchSuffix
-	cmd:gettext$secondaryArchSuffix
+	gettext${secondaryArchSuffix}
```

Comment:
Why was this changed?

Reply:
Didn't see a requirement for the command usage.

Reply:
Did you check configure output? I'm pretty sure it is found in there.

Reply:
Yes... it worked fine.

Reply:
Let me be much more clear here my my response - thanks to begasus and OscarL:
'gettext - not found situation' - yes, I'm aware of it. Otherwise, seems to find what it needs (like: msgfmt) and builds without issue. Functionally, the shell worked for me - but will need more review time as I looking into the unit tests.